src/arch/helperavx512f.h

```diff
@@ -207,6 +207,8 @@ static INLINE vopmask visnan_vo_vd(vdouble d) {
 
 static INLINE vint vilogbk_vi_vd(vdouble d) { return vrint_vi_vd(_mm512_getexp_pd(d)); }
 
+static INLINE vint vilogb2k_vi_vd(vdouble d) { return vrint_vi_vd(_mm512_getexp_pd(d)); }
```

Comment:
vilogb2k_vi_vd has a relaxed specification than vilogbk_vi_vd. For AVX512F target, the optimal codes happened to be same.